app/views/posts/index.html.erb

```diff
@@ -1,12 +1,22 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Posts</title>
-  </head>
-  <body>
-    <h1>Here is a list of the posts shown for a designated user.</h1>
-  </body>
-</html>
\ No newline at end of file
+<div id="user-container">
+   <%= render partial: "users/user_info", locals: { user: @user } %>
+</div>
+
+<ul id="post-container">
+  <% @posts.each_with_index do |post, i| %>
+    <li>
+      <%= link_to "/users/#{@user.id}/posts/#{post.id}", id: "post" do %>
+        <%= render partial: "posts/post_info", locals: { post: post } %>
+      <% end %>
+      <ul id="comment-container">
+        <% post.five_most_recent_comments.each do |comment| %>
+          <li id="comment">
+            <p><%= comment.user.name %>: <%= comment.text %></p>
+          </li>
+        <% end %>
+      </ul>
+    </li>
```

Comment:
- [ ] **[OPTIONAL]** There is a lot of blank space between the user info and the posts' listings. Kindly minimize it to match the wireframe.

  ![image](https://github.com/katarighe/blog-app/assets/81468675/ea60fb8b-2ea3-497d-9381-66ae2422c791)

  ![image](https://github.com/katarighe/blog-app/assets/81468675/946d6885-fb33-4eb2-83be-efb8b2cfe657)